packages/core/src/Stores/notification-store.js

```diff
@@ -250,7 +254,10 @@ export default class NotificationStore extends BaseStore {
             }
         } else if (has_mt5_account_with_rejected_poa) {
             this.addNotificationMessage(this.client_notifications.poa_rejected_for_mt5);
-        } else if (!['none', 'pending', 'expired'].includes(document.status)) {
+        } else if (
+            !['none', 'pending', 'expired'].includes(document.status) ||
+            status.includes('allow_poa_resubmission')
+        ) {
```

Comment:
in case of POI POA not-submitted/ failed/ expired are we showing one common notification ? or in the notification it will state "POA/POI pending/ failed/ expired in the notification?

Reply:
We have  specific notification messages for those status. I'm not sure why we have this specific condition